index.md

```diff
@@ -54,7 +54,7 @@ How to work with the Umbraco codebase.
 Cheatsheets for easy overview of different areas like the Razor API and Views syntax.
 
 ### [Umbraco Cloud](Umbraco-Cloud/)
-How to use Umbraco Cloud: Get started, set up your Umbraco Cloud project, deploy and troubleshoot issues.
+How to use Umbraco Cloud: Get started, set up your Umbraco Cloud project and deploy and troubleshoot issues.
```

Comment:
How about instead we say something like: 
_How to use Umbraco Cloud: Getting started, setting up your Umbraco Cloud project, learn to use the deployment approaches and troubleshoot issues._ ?